pypit/arvcorr.py

```diff
@@ -1,395 +1,78 @@
+from __future__ import (print_function, absolute_import, division, unicode_literals)
```

Comment:
Since this script is now significantly reduced (compared to before), it might make sense to simply put these functions in `arwave.py`, and delete `arvcorr.py`.